[2024-04-08T12:05:01.476+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.get_files manual__2024-04-08T12:04:57.665881+00:00 [queued]>
[2024-04-08T12:05:01.485+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.get_files manual__2024-04-08T12:04:57.665881+00:00 [queued]>
[2024-04-08T12:05:01.486+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-08T12:05:01.509+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): get_files> on 2024-04-08 12:04:57.665881+00:00
[2024-04-08T12:05:01.519+0000] {standard_task_runner.py:60} INFO - Started process 169948 to run task
[2024-04-08T12:05:01.522+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl', 'get_files', 'manual__2024-04-08T12:04:57.665881+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmp6vgb1use']
[2024-04-08T12:05:01.524+0000] {standard_task_runner.py:88} INFO - Job 46: Subtask get_files
[2024-04-08T12:05:01.594+0000] {task_command.py:423} INFO - Running <TaskInstance: etl.get_files manual__2024-04-08T12:04:57.665881+00:00 [running]> on host 8056db226622
[2024-04-08T12:05:01.711+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='get_files' AIRFLOW_CTX_EXECUTION_DATE='2024-04-08T12:04:57.665881+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-08T12:04:57.665881+00:00'
[2024-04-08T12:05:01.714+0000] {logging_mixin.py:188} INFO - 2 files found in /opt/***/dags/data
[2024-04-08T12:05:01.714+0000] {python.py:202} INFO - Done. Returned value was: ['/opt/***/dags/data/github_events_02.json', '/opt/***/dags/data/github_events_01.json']
[2024-04-08T12:05:01.746+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=etl, task_id=get_files, execution_date=20240408T120457, start_date=20240408T120501, end_date=20240408T120501
[2024-04-08T12:05:01.780+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-08T12:05:01.803+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
